src/components/page-layout/Layout.js

```diff
@@ -12,34 +12,37 @@ import { useStaticQuery, graphql } from 'gatsby';
 // Components
 import Header from './Header';
 import NavigationBar from './NavigationBar';
+import GrocerySection from '../grocery-view/grocery-section/GrocerySection';
 
 // Styles
 import styles from './styles/layout.module.css';
 
 const Layout = ({ children }) => {
   const data = useStaticQuery(graphql`
-        query SiteTitleQuery {
-            site {
-                siteMetadata {
-                    title
-                }
-            }
+    query SiteTitleQuery {
+      site {
+        siteMetadata {
+          title
         }
-    `);
+      }
+    }
+  `);
 
   return (
-        <>
-            <Header siteTitle={data.site.siteMetadata.title} />
-            <NavigationBar />
-            <div className={styles.body}>
-                <main>{children}</main>
-                <footer>
-                    © {new Date().getFullYear()}, Built with
-                    {' '}
-                    <a href="https://www.gatsbyjs.org">Gatsby</a>
-                </footer>
-            </div>
-        </>
+    <>
+      <Header siteTitle={data.site.siteMetadata.title} />
+      <NavigationBar />
+      <div className={styles.body}>
+        <main>
+          {children}
+          <GrocerySection />
+        </main>
+        <footer>
+          © {new Date().getFullYear()}, Built with{' '}
+          <a href="https://www.gatsbyjs.org">Gatsby</a>
+        </footer>
+      </div>
+    </>
```

Comment:
This is something I was wondering about and we should figure out.

The way I see it, we have two options: 
1. We do it this way, where there is really only a single page and the GrocerySection exists within the layout. The problem is that I don't see what we'll do with the Receipt.
2. We create two pages: index.js (which will show groceries) and receipt.js (which will show the receipt)

If we go with #2 (and I think we should), then we ought to remove <GrocerySection> from the layout component and insstead add it to src/pages/index.js     and the layout should just consist of the nav.

Reply:
Agreed, #2 is probably the way to go